packages/block-editor/src/hooks/border.js

```diff
@@ -39,22 +57,80 @@ export function BorderPanel( props ) {
 		return null;
 	}
 
+	const defaultBorderControls = getBlockSupport( props.name, [
+		BORDER_SUPPORT_KEY,
+		'__experimentalDefaultControls',
+	] );
+
+	const createResetAllFilter = (
+		borderAttribute,
+		topLevelAttributes = {}
+	) => ( newAttributes ) => ( {
+		...newAttributes,
+		...topLevelAttributes,
+		style: {
+			...newAttributes.style,
+			border: {
+				...newAttributes.style?.border,
+				[ borderAttribute ]: undefined,
+			},
+		},
+	} );
+
 	return (
-		<InspectorControls>
-			<PanelBody
-				className="block-editor-hooks__border-controls"
-				title={ __( 'Border' ) }
-				initialOpen={ false }
-			>
-				{ ( isWidthSupported || isStyleSupported ) && (
-					<div className="block-editor-hooks__border-controls-row">
-						{ isWidthSupported && <BorderWidthEdit { ...props } /> }
-						{ isStyleSupported && <BorderStyleEdit { ...props } /> }
-					</div>
-				) }
-				{ isColorSupported && <BorderColorEdit { ...props } /> }
-				{ isRadiusSupported && <BorderRadiusEdit { ...props } /> }
-			</PanelBody>
+		<InspectorControls __experimentalGroup="border">
+			{ isWidthSupported && (
+				<ToolsPanelItem
+					className="single-column"
```

Comment:
I'm still catching up with the changes happened in the past 2 weeks, but I believe that we're moving away from the `single-column` className?

Reply:
We did in the `ToolsPanel` itself. 

However, we still need a means of managing which controls span what columns. I did have a PR switching the `ToolsPanel` to expose grid or grid item properties to manage the layout. It was [requested](https://github.com/WordPress/gutenberg/pull/35621#issuecomment-946071523)  to avoid that for now until more discussion had taken place around just exactly what props we wished to expose.

I believe @stacimc has split the updating of border panel layout into a new PR ( https://github.com/WordPress/gutenberg/pull/35938 ) rather than updating this one directly. This `single-column` class name is a remnant from the old approach in the `ToolsPanel` however it is used to make those controls only span a single column in this panel [here](https://github.com/WordPress/gutenberg/pull/35938/files#diff-32de6bc912076f3b778ba2b7a6cafcc835596c521715d0bc644618fd8777b80eR1-R5).

Reply:
This makes sense! I think I got confused by the fact that the className is the same — but it looks like the styles now come from _outside_ the `ToolsPanel` component. All good!